service/switchboardapp/contracts/api-spec.yaml

```diff
@@ -11,7 +11,7 @@ servers:
 paths:
   /Encoders:
     get:
-      operationId: getEncoders
+      operationId: retrieveAllEncoders
```

Comment:
i changed the ids  to match the method names we had, i also changed other stuff. I dont know the best conventions for the api-spec document( please advise if theres something wrong), however i made some changes so that it atleast matches our current code. I will implement it this pr if needed